src/traces/histogram2d/index.js

```diff
@@ -15,6 +15,7 @@ Histogram2D.attributes = require('./attributes');
 Histogram2D.supplyDefaults = require('./defaults');
 Histogram2D.calc = require('../heatmap/calc');
 Histogram2D.plot = require('../heatmap/plot');
+Histogram2D.layerName = 'heatmaplayer';
```

Comment:
**N.B.** Trace modules that reuse a plot method verbatim from another module need to declare `layerName` so that their traces are combined into the same trace module layer.

Reply:
.... there's probably a what to :hocho: `layerName` entirely by building some trace-to-layerName map object within the `Registry`, but oh well I didn't think adding a few of :arrow_heading_up: would be too bad. Thoughts?

Reply:
I like it this way - explicit and clear.